apex/contrib/optimizers/distributed_fused_adam.py

```diff
@@ -709,6 +730,7 @@ def __init__(
         # Determine bucket sizes
         dtype_size = torch.finfo(self.grad_sync_dtype).bits // 8
         self.alignment: int = 128 // dtype_size
+        self.bucket_cap_mb: float = bucket_cap_mb
```

Comment:
Nit: I don't think `bucket_cap_mb` is used.
```suggestion
```
If we want to use it externally, we could recompute something more accurate:
```python
bucket_size = self.default_shard_size * self.distributed_size
self.default_bucket_size_mb: float = bucket_size * dtype_size / (1024 * 1024)
```

Reply:
Sorry but I have to revert this. `self.bucket_cap_mb` is indeed not used anymore in the code, but it's helpful for use to ensure all Dist Adam arguments are correct. There's a code snippet to print Dist Adam with all important arguments in `__repr__`, `self.bucket_cap_mb` is used there. So I think we need to keep it.